File: exo2/exo2.java
import java.util.*;

public class exo2{
	public static void main(String[] args){
int[] tableau = {3,20,15,28,9,6,52};
int permut = 0;
boolean sorted = false;
boolean moved = false;
int i =0;

while (sorted == false){
	moved = false;
	i = 0;

	while (i<tableau.length-1){
		if(tableau[i+1] > tableau[i]){
			
			permut = tableau[i]; 
			tableau[i] = tableau[i + 1]; 
			tableau[i + 1] = permut; 
			moved = true; 


		}	
		i += 1;


	}
System.out.println(Arrays.toString(tableau));
	if (moved == false){
		sorted = true;
	}
}

}
}

File: exo2/exo2.php
<?php
$tableau = array(3,25,1,6,23,56);	# On declare le tableau
$permut = 0;				# et les variables.
$sorted = false;
	while ($sorted == false) {
		$sorted = true;
		for ($i = 0; $i < count($tableau)-1; $i++) {
#tant que l'index courant est inferieur a la longueur du tableau
# si la valeur suivant est plus grande, on switch
		if ($tableau[$i] < $tableau[$i + 1]) {
			$permut = $tableau[$i];
			$tableau[$i] = $tableau[$i + 1];
			$tableau[$i + 1] = $permut;
			$sorted = false;
		print_r ($tableau);
          }
      }
  }
  print_r ($tableau);
?>
